layouts/partials/opengraph.html

```diff
@@ -0,0 +1,11 @@
+{{ with .Site.Params.title }}
+<meta property="og:title" content="{{ . }}" />
+{{ end }}
```

Comment:
Can be simplified to `<meta property="og:title" content="{{ .Site.Params.title }}" />`